### DISCUSSION TITLE
user_data_source paramter in edit_list.conf

### DISCUSSION THREAD

**Discussion by UAProfessorProton** [sympa-community/sympa#1896] [Q&A]
Hello, just want to get reliable information to confirm if user_data_source paramters is already removed/deprecated for edit_list.conf. 

We are attempting to have these lines included in our edit_list.conf file. When logged in as a list owner, and not a listmaster, I still see data sources still accessible to me. 

user_data_source                owner,privileged_owner          hidden

include_file                    owner,privileged_owner          hidden

include_remote_file             owner,privileged_owner          hidden

include_list                    owner,privileged_owner          hidden

include_sql_query               owner,privileged_owner          hidden

include_ldap_query              owner,privileged_owner          hidden

include_ldap_2level_query       owner,privileged_owner          hidden

include_remote_sympa_list       owner,privileged_owner          hidden

include_admin                   owner,privileged_owner          hidden

**Comment by ikedas** [comment]
Hi @UAProfessorProton ,
Perhaps you are a _privileged_ owner, not a normal one.

**Reply by UAProfessorProton** [reply]
@ikedas, thank you for your reply, does this mean if your account doesn't fall to any of the roles like the line below, that parameter will be hidden? Is that correct? 

user_data_source owner,privileged_owner hidden

**Comment by ikedas** [comment]
@UAProfessorProton , it's hard to answer your question because it is not clear what setting you are actually trying.
* If you have customized `edit_list.conf` or any web templates, revert them and try again. Or the results will not be what you would expect from the default settings.
* Make sure you are a normal owner, privileged owner, or listmaster (you can be more than one at the same time).If you have not made customization above, it is unlikely that you are "logged in as a list owner, and not a listmaster".

**Reply by UAProfessorProton** [reply]
Thank you @ikedas, we found the cause of the issue. Apparently we didn't have the member_include parameter added to our edit_list.conf ,where, should be configured hidden when you're an owner or privileged owner.  

Problem solved. :-)
